### PR TITLE
minor text changes 

### PR DESCRIPTION
**Pull request by katvanderahe**

### CODE REVIEW

**Comment by jeffreycwitt** [comment]
Awesome. Keep 'em coming!

**Review by jeffreycwitt — COMMENTED** [review]
This looks great to me.

File: bHY6yh-d1e400/cod-pu6y7d_bHY6yh-d1e400.xml [hunk 1]
```diff
@@ -74,14 +74,14 @@
...
                   <lb ed="#V" n="13"/>scientia colendorum deorum. </quote>
             </p>
             <p xml:id="bHY6yh-d1e400-d1e158"> ¶
-               <quote xml:id="bHY6yh-d1e400-Qd1e160"> Idem rethori
-                  <lb ed="#V" n="14" break="no"/>corum artis vereris lib. 2. c. 9. Religio est virtus
+               <quote xml:id="bHY6yh-d1e400-Qd1e160"> <ref>Idem rethori
+                  <lb ed="#V" n="14" break="no"/>corum artis veteris lib. 2. c. 9.</ref> Religio est virtus
                   <lb ed="#V" n="15"/>que superioris cuiusdam nature: quam diuinam
-                  <lb ed="#V" n="16"/>notant curam: cerimoniamquem affert. </quote>
+                  <lb ed="#V" n="16"/>notant curam: cerimoniamque affert. </quote>
```
Comment (ChrisNighman) [inline bHY6yh-d1e400/cod-pu6y7d_bHY6yh-d1e400.xml:84]: OK, looks good

**Review by ChrisNighman — APPROVED** [review]

**Comment by katvanderahe** [comment]
<quote xml:id="bHY6yh-de1319-Qd1e335">Infernus et perditio coram domino:
       <lb ed="#V" n="21"/>quanto magis corda filiorum hominum. </quote> lines 167-8
Isn’t attributed as with a ref in the text but I did some digging and is from Proverbs 15:11 and I wasn’t sure what to do with that or if anything needed to be done I just wanted to make a note of it 
Also can issue with 
<quote xml:id="bHY6yh-de1319-Qd1e480">Unumquemque iuxta vi
       <lb ed="#V" n="51"/>as suas iudicabo ait dominus deus. </quote>
Also with 
<quote xml:id="bHY6yh-de1319-Qd1e523">¶ Adiecit mandata et praecepta sua. Si volue
       <lb ed="#V" n="60" break="no"/>ris mandata conseruare: confuabunt te etc. </quote>

I just wanted to make a note of these

**Pull request merged**